### PR TITLE
Feat (package-generator): Add label experimental to typescript in package generator cli

### LINKED ISSUE #123: Mark "TypeScript" as "experimental" option
## Provide a description of the task

When the tool asks about the language, `TypeScript` should be marked as _experimental_.

image

```
Choose your programming language:
* JavaScript
* TypeScript (experimental)
```

### PR DESCRIPTION
**Pull request by martnpaneq**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Internal: Added label "experimental" to TypeScript in package generator CLI. Closes #123.

### CODE REVIEW

File: packages/ckeditor5-package-generator/lib/utils/choose-programming-language.js [hunk 1]
```diff
@@ -8,8 +8,8 @@
 const { prompt } = require( 'inquirer' );
 
 const PROGRAMMING_LANGUAGES = {
-	JavaScript: 'js',
-	TypeScript: 'ts'
+	JavaScript: { value: 'js', displayName: 'JavaScript' },
+	TypeScript: { value: 'ts', displayName: 'TypeScript (experimental)' }
```
Comment (pomek) [inline packages/ckeditor5-package-generator/lib/utils/choose-programming-language.js:12]: Shouldn't it be an array, then?

Comment (martnpaneq) [reply packages/ckeditor5-package-generator/lib/utils/choose-programming-language.js]: I was a little bit worried at the beginning that changing it to an array might break something. But all works fine. Good catch!

**Review by pomek — APPROVED** [review]
LGTM.

**Review by psmyrek — APPROVED** [review]
LGTM 🎉

**Pull request merged**